app/jobs/get_usps_proofing_results_job.rb

```diff
@@ -39,9 +40,19 @@ def perform(_now)
 
     check_enrollments(enrollments)
 
+    percent_enrollments_errored = 0
+    if enrollment_outcomes[:enrollments_checked] > 0
+      percent_enrollments_errored =
+        (enrollment_outcomes[:enrollments_errored].fdiv(
+          enrollment_outcomes[:enrollments_checked],
+        ) * 100).round
```

Comment:
Given the AC I think [`ceil`](https://ruby-doc.org/core-3.0.0/Float.html#method-i-ceil) would provide the more accurate end result, but this is probably ok given the current goal. Personally I would be inclined to [`round`](https://ruby-doc.org/core-3.0.0/Float.html#method-i-round) to two decimal places instead of 0.

The current implementation does mean that if we have something like 10.1% which should technically trigger the alarm based on AC that it would not, but the threshold is a little arbitrary so I think this deviation is within acceptable bounds.

Here's what you could do to round to two places instead & make this a little more precise:
```suggestion
        ) * 100).round(2)
```

Reply:
Done